docs/data/toolpad/core/components/dashboard-layout/dashboard-layout.md

```diff
@@ -8,22 +8,35 @@ components: AppProvider, DashboardLayout
 
 <p class="description">The dashboard layout component provides a customizable out-of-the-box layout for a typical dashboard page.</p>
 
+The `DashboardLayout` component is a quick, easy way to provide a standard full-screen layout with a header and sidebar to any dashboard page, as well as ready-to-use and easy to customize navigation and branding.
+
+Many features of this component are configurable through the [AppProvider](https://mui.com/toolpad/core/react-app-provider/) component that should wrap it.
```

Comment:
To follow-up outside of this PR: 
Would it make sense to provide all the available functionality that is read from the app provider as props as well? That way the suer has the choice to use with app provider and stand alone. If we let the props fall-back to the app provider contexts we have a way to offer following functionalities:
- use with app provider
- use standalone with the props
- use with app provider but overwrite specific context

Reply:
Yeah I was planning to do that next and then edit the docs accordingly, we can prioritize the props that are passed otherwise default to the app provider.